contexts/DonationContext/src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -64,6 +64,10 @@ public function addDonation( AddDonationRequest $donationRequest ): AddDonationR
 			$donation->markForModeration();
 		}
 
+		if ( $this->policyValidator->isAutoDeleted( $donationRequest ) ) {
+			$donation->markAsDeleted();
```

Comment:
For donations we do not use bits to reflect the status. Marking for moderation and subsequently marking as deleted would result in `status` being `D`.

Although @TillWMDE said to keep it simple, I'm slightly confused about what should actually happen here.